### ISSUE TITLE
 java.lang.NullPointerException at org.osmdroid.config.DefaultConfigurationProvider.load

### ISSUE THREAD

**Issue by mh-** [mh-/corona-warn-companion-android#84]
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3270)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3409)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:83)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2016)
  at android.os.Handler.dispatchMessage (Handler.java:107)
  at android.os.Looper.loop (Looper.java:214)
  at android.app.ActivityThread.main (ActivityThread.java:7356)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:491)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:940)
Caused by: java.lang.NullPointerException: 
  at org.osmdroid.config.DefaultConfigurationProvider.load (DefaultConfigurationProvider.java:337)
  at org.tosl.coronawarncompanion.MainActivity.onCreate (MainActivity.java:237)
  at android.app.Activity.performCreate (Activity.java:7824)
  at android.app.Activity.performCreate (Activity.java:7813)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1306)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3245)
```

(report from Google Play; 4 users reported a total of 25 occurrences, 1 user reported 22 occurrences)

@uhengart you might want to have a look...

**Comment by mh-** [comment]
Probably related:
https://github.com/osmdroid/osmdroid/issues/1621
https://github.com/osmdroid/osmdroid/issues/1628

**Comment by uhengart** [comment]
Thanks for letting me know.

I don't see this crash on my device. After looking at DefaultConfigurationProvider.load() in osmdroid 1.6.6 and 1.6.8, I'm pretty sure that the crash is caused by the same problem as in your two references above. The (temporary) fix is to downgrade osmdroid to 1.6.6.. A patch that should fix the problem is already in the osmdroid repo. We can upgrade to 1.6.9 when it's out.

**Comment by mh-** [comment]
Related to #88
